test/unit/registry.test.ts

```diff
@@ -128,6 +135,29 @@ describe('Registry utilities', () => {
       fs.rmdirSync(`deployments/warp_routes/${MOCK_SYMBOL}`);
     }).timeout(5_000);
   }
+
+  describe('MergedRegistry', async () => {
+    it('Merges metadata from multiple registries', async () => {
+      const mergedMetadata = await mergedRegistry.getMetadata();
+      const localMetadata = await localRegistry.getMetadata();
+      expect(mergedMetadata['ethereum'].chainId).to.eql(1);
+      // Confirm the partial registry metadata is merged
+      expect(mergedMetadata['ethereum'].displayName).to.eql(MOCK_DISPLAY_NAME);
+      expect(localMetadata['ethereum'].displayName).to.not.eql(MOCK_DISPLAY_NAME);
+      // Confirm other chains are not affected
+      expect(mergedMetadata['arbitrum']).to.eql(localMetadata['arbitrum']);
```

Comment:
nit: maybe confirm all other chains are not affected

Reply:
I'll opt for keeping it limited to arbitrum because it's better if this tests doesn't assume a lot of overlap btwn the local registry and the canonical github one (since people will sometimes fork the registry)